eduNEXT/edunext-platform
# fix: if pages and resources view is disabled, show all pages in studi…

## Enable Studio Pages show/hide

The ability to see and enable/disable wiki and progress tabs was removed from studio along with the ability to re-order non-static tabs. The ability to toggle the Wiki tab was moved to the pages and resources section of the course authoring MFE. If that MFE is unavailable this means there is no way to show/hide the tabs. This reverts some of the old changes if the pages and resources view is disabled.

## Description

This PR is a backport for https://github.com/openedx/edx-platform/commit/8169aa99da32902b00a7f76a20c627ea1d44576c

## Supporting information

This is part of the reported migration issues [JIRA DS-348](https://edunext.atlassian.net/browse/DS-348)

## Testing instructions

- Use this branch in your development environment. 
- Add this feature to `cms.env`

``` yml
FEATURES:
       ALLOW_HIDING_DISCUSSION_TAB: true
```

- Go to Studio > your course > Content > Pages and check the disable switch is enabled in progress and discussion pages.

Is available in the nuez closter access here https://studio.nutmeg-live.edunext.link/tabs/course-v1:edX+1+1
